### PR TITLE
rbd: return error if we specified a wrong image name for rbd du

### PR DESCRIPTION
**Pull request by yangdongsheng**
Fixes: http://tracker.ceph.com/issues/16987
Signed-off-by: Dongsheng Yang <dongsheng.yang@easystack.cn>

### CODE REVIEW

File: src/tools/rbd/action/DiskUsage.cc [hunk 1]
```diff
@@ -206,6 +208,10 @@ static int do_disk_usage(librbd::RBD &rbd, librados::IoCtx &io_ctx,
       ++count;
     }
   }
+  if (!found) {
+    std::cerr << "specified image of " << imgname << "is not found." << std::endl;
+    return -EINVAL;
```
Comment (vshankar) [inline src/tools/rbd/action/DiskUsage.cc]: -ENOENT?

File: src/tools/rbd/action/DiskUsage.cc [hunk 2]
```diff
@@ -206,6 +208,10 @@ static int do_disk_usage(librbd::RBD &rbd, librados::IoCtx &io_ctx,
       ++count;
     }
   }
+  if (!found) {
+    std::cerr << "specified image of " << imgname << "is not found." << std::endl;
```
Comment (dillaman) [inline src/tools/rbd/action/DiskUsage.cc]: Minor: remove "of" and add extra space before "is not found".

Comment (yangdongsheng) [inline src/tools/rbd/action/DiskUsage.cc]: Okey

File: src/tools/rbd/action/DiskUsage.cc (hunk 1 above)
Comment (yangdongsheng) [inline src/tools/rbd/action/DiskUsage.cc]: Good suggestion

File: src/tools/rbd/action/DiskUsage.cc (hunk 2 above)
Comment (yangdongsheng) [inline src/tools/rbd/action/DiskUsage.cc]: @dillaman this is already updated, but I am not sure why it's still expanded here in github.

**Comment by dillaman** [comment]
lgtm

**Pull request merged**